autoload/neomake/makers/ft/python.vim

```diff
@@ -53,9 +51,18 @@ function! neomake#makers#ft#python#flake8()
 endfunction
 
 function! neomake#makers#ft#python#Flake8EntryProcess(entry)
-    if a:entry.type ==# 'F'
-        let a:entry.type = 'E'
+    if a:entry.type ==# 'F'  " PyFlake errors
+        let type = 'E'
+    elseif a:entry.type ==# 'E' || a:entry.type ==# 'W'  " PEP8 errors & warnings
+        let type = 'W'
+    elseif a:entry.type ==# 'N' || a:entry.type ==# 'D'  " Naming (PEP8) & docstring (PEP257) conventions
+        let type = 'W'
+    elseif a:entry.type ==# 'C' || a:entry.type ==# 'T'  " mccabe complexity & todo notes
+        let type = 'I'
+    else
+        let type = 0
```

Comment:
Why not empty string instead? If the types aren't consistent vimscript can get sad.